pages/license.md

```diff
@@ -2,60 +2,24 @@
-        5.  12.8.5. may not be conditioned on payment of royalties, fees or other consideration;
-        6.  12.8.6. may be suspended with respect to any licensee when licensor issued by licensee for infringement of claims essential to implement the Specification or any W3C Recommendation;
-        7.  12.8.7. may not impose any further conditions or restrictions on the use of any technology, intellectual property rights, or other restrictions on behavior of the licensee, but may include reasonable, customary terms relating to operation or maintenance of the license relationship such as the following: choice of law and dispute resolution;
-        8.  12.8.8. shall not be considered accepted by an implementer who manifests an intent not to accept the terms of the W3C CLA RF Licensing Requirements license as offered by the licensor.
-        9.  12.8.9. The RF license conforming to the requirements in this policy shall be made available by the licensor as long as the Specification is in effect. The term of such license shall be for the life of the patents in question.
-
-        I am encouraged to provide a contact from which licensing information can be obtained and other relevant licensing information. Any such information will be made publicly available.
-
-    9.  12.9. You or Your. "You," "you," or "your" means any person or entity who exercises copyright or patent rights granted under this CLA, and any person or entity that person or entity controls.
+## W3C SOFTWARE AND DOCUMENT NOTICE AND LICENSE
+
+This work is being provided by the copyright holders under the following license.
+
+### License
+
+By obtaining and/or copying this work, you _(the licensee)_ agree that you have read, understood, and will comply with the following terms and conditions.
+
+Permission to **copy**, **modify**, and **distribute** this work, with or without modification, for any purpose and without fee or royalty is hereby granted, provided that you include the following on **ALL** copies of the work or portions thereof, including modifications:
+
+- The full text of this **NOTICE** in a location viewable to users of the redistributed or derivative work.
+- Any pre-existing intellectual property disclaimers, notices, or terms and conditions. If none exist, the [W3C](https://www.w3.org/) Software and Document Short Notice should be included.
+- Notice of any changes or modifications, through a copyright statement on the new code or document such as "This software or document includes material copied from or derived from [title and URI of the W3C document]. Copyright © [YEAR] W3C® (MIT, ERCIM, Keio, Beihang)."
+
+### Disclaimer
+
+THIS WORK IS PROVIDED "AS IS," AND COPYRIGHT HOLDERS MAKE NO REPRESENTATIONS OR WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO, WARRANTIES OF MERCHANTABILITY OR FITNESS FOR ANY PARTICULAR PURPOSE OR THAT THE USE OF THE SOFTWARE OR DOCUMENT WILL NOT INFRINGE ANY THIRD PARTY PATENTS, COPYRIGHTS, TRADEMARKS OR OTHER RIGHTS.
```

Comment:
Same here. I suspect this needs to be upper case, but then let's use CSS for that.